sympy/polys/tests/test_rings.py

```diff
@@ -1359,10 +1359,46 @@ def test_PolyElement_pdiv():
     q, r = x + y, 0
 
     assert f.pdiv(g) == (q, r)
-    assert f.prem(g) == r
     assert f.pquo(g) == q
     assert f.pexquo(g) == q
 
+
+def test_PolyElement_coeff_wrt():
+
+    R, x, y, z = ring("x, y, z", ZZ)
+
+    p = 4*x**3 + 5*y**2 + 6*y**2*z + 7
+    assert p.coeff_wrt(1, 2) == 6*z + 5 # using generator index
+    assert p.coeff_wrt(x, 3) == 4 # using generator
+
+    p1 = 2*x**4 + 3*x*y**2*z + 10*y**2 + 10*x*z**2
+    assert p1.coeff_wrt(x, 1) == 3*y**2*z + 10*z**2
+    assert p1.coeff_wrt(y, 2) == 3*x*z + 10
+
+    p2 = 4*x**2 + 2*x*y + 5
+    raises(ValueError, lambda: p2.coeff_wrt(z, 1))
+    raises(ValueError, lambda: p2.coeff_wrt(y, 2))
```

Comment:
Should these not just return zero?

Reply:
Yes, I think it should return zero
now it is giving this error 
```python
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\DELL\OneDrive\Desktop\gsoc_pr2\sympy\polys\rings.py", line 2574, in coeff_wrt
    terms = [(m, c) for m, c in p.iterterms() if m[i] == deg]
    ^^^^^^^^^^^^^^
ValueError: not enough values to unpack (expected 2, got 0)
```

Reply:
Then it should be fixed to return zero.